ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml

```
@@ -1,18 +1,22 @@
 ref:
   as: ipi-conf-aws
   from_image:
-   namespace: origin
-   name: centos
-   tag: '8'
+    namespace: origin
+    name: centos
+    tag: "8"
   commands: ipi-conf-aws-commands.sh
   resources:
     requests:
       cpu: 10m
       memory: 100Mi
   env:
     - name: COMPUTE_NODE_TYPE
-      default: 'null'
+      default: "null"
       documentation: |-
         The instance type to use for compute nodes (e.g. AWS https://aws.amazon.com/ec2/instance-types/)
+    - name: ZONE
+      default: 2
```

```suggestion
      default: "2"
```

since i think steps require string inputs.